Carbon/navigation/screens/Main/MainContainer.js

```diff
@@ -115,16 +110,16 @@ const ForumStack = ({ navigation }) => {
                 name={'Quiz'}
                 component={QuizScreen}
                 options={{
-                   // headerShown: false, // Set to false for now until we need to implement headers for this screen
-                   headerShown: true,
+                    // headerShown: false, // Set to false for now until we need to implement headers for this screen
+                    headerShown: true,
                 }}
             />
             <Stack.Screen
                 name={'Browser'}
                 component={BrowserScreen}
                 options={{
-                   // headerShown: false, // Set to false for now until we need to implement headers for this screen
-                   headerShown: true,
+                    // headerShown: false, // Set to false for now until we need to implement headers for this screen
```

Comment:
I'm wondering if we should create a tag for items we need to come back to once changes are implemented that we need in order to enable/disable certain things.